java/daikon/chicory/ChicoryPremain.java

```diff
@@ -44,6 +44,18 @@
    **/
   private static boolean doPurity = false;
 
+  // This @SuppressWarnings is for the two lines
```

Comment:
I suggest abstracting the two lines into their own method so that the scope of the @SuppressWarnings can be smaller.